.gitignore

```diff
@@ -11,6 +11,9 @@
 # User-specific files (MonoDevelop/Xamarin Studio)
 *.userprefs
 
+# Project Launch Settings
```

Comment:
Why excluding launchSettings from source control?

Reply:
It is because because whenever I reload the solution, all the launch settings automatically marked as changed to but diff shows there is absolutely nothing changed.

However you can ignore this and include those in the source control you that is needed.